### ISSUE TITLE
Install fails

### ISSUE THREAD

**Issue by julia-string** [dougludlow/ng2-bs3-modal#5]
When running `npm install ng2-bs3-modal` I get the following error.
Is there anything I'm doing wrong?
```
npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "ng2-bs3-modal"
npm ERR! node v5.6.0
npm ERR! npm  v3.7.2
npm ERR! code ELIFECYCLE

npm ERR! ng2-bs3-modal@0.1.1 postinstall: `npm run peer`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ng2-bs3-modal@0.1.1 postinstall script 'npm run peer'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the ng2-bs3-modal package,
npm ERR! not with npm itself.
```

[npm-debug.log.zip](https://github.com/dougludlow/ng2-bs3-modal/files/134686/npm-debug.log.zip)

**Comment by dougludlow** [comment]
I'm getting the same thing, weird. Looking into it now.

**Comment by julia-string** [comment]
Ah I see, thank you!

**Comment by dougludlow** [comment]
@julia-string should be good now. Give it a try and let me know how it goes.

**Comment by julia-string** [comment]
You're a star! Thank you!

**Comment by dougludlow** [comment]
:+1:

**Comment by julia-string** [comment]
Sorry, one more thing: I get `UNMET PEER DEPENDENCY bootsrap@3.3` but I'm running on bootstrap@3.3.6.

**Comment by dougludlow** [comment]
I've recently added the peer dependency and I may have botched the semver syntax. Lemme take look into that as well.

**Comment by dougludlow** [comment]
Oh man... I fat fingered bootstrap... had it in there as bootsrap. Thanks for catching my mistakes! Should be good to go in v0.1.3.

Thanks again!

**Comment by julia-string** [comment]
Cool! That worked now, thank you! 
Sorry to be such a bugger. The import path still seems wrong, it doesn't find the correct file when using `import { MODAL_DIRECTIVES } from 'ng2-bs3-modal';` but instead returns `ng2-bs3-modal:1 Uncaught SyntaxError: Unexpected token <`. I tried different paths also.

**Comment by dougludlow** [comment]
hmm... try `import { MODAL_DIRECTIVES } from "ng2-bs3-modal/dist/ng2-bs3-modal";`

**Comment by julia-string** [comment]
Yeah, I tried that too, but then it throws the same error (likely a different cause), when including the directive `MODAL_DIRECTIVES`.

**Comment by dougludlow** [comment]
@julia-string how are you transpiling/loading your scripts? systemjs?

**Comment by rsnider19** [comment]
I might be having the same issue as @julia-string . I have `import {MODAL_DIRECTIVES} from 'ng2-bs3-modal/dist/ng2-bs3-modal';` and am using SystemJS, but I am getting a 404 when the browser looks for `http://localhost:12345/ng2-bs3-modal/dist/ng2-bs3-modal`

**Comment by dougludlow** [comment]
@rsnider19 it sounds like I need to take a better look and the npm install flow.

**Comment by rsnider19** [comment]
Thanks! In the meantime, I think I'll just add the components to my project so that I can start using them.

**Comment by dougludlow** [comment]
:+1:

**Comment by nhhockeyplayer** [comment]
Nice share dude... love the way you manhandled (implements) that interface deep within angular2 like a JAVA PRO. Time to step into the fast lane...   : )

**Comment by Flea505** [comment]
@dougludlow Any progress on the "SyntaxError: Unexpected token < " when using the following import... `import {MODAL_DIRECTIVES} from 'ng2-bs3-modal/dist/ng2-bs3-modal';`??

**Comment by dougludlow** [comment]
@Flea505 none, yet... I'm on a family vacation right now. But I'll probably have some time to look into it this weekend.

**Comment by dougludlow** [comment]
This should be fixed now in 0.1.4. Please give it a go and let me know if you have any issues.

You'll need to change the import path to the following:

    import { MODAL_DIRECTIVES } from "ng2-bs3-modal/ng2-bs3-modal";

Also, if you're loading your scripts manually (not using something like systemjs), I've added `bundles/ng2-bs3-modal.js` which you can include directly from node_modules, if desired.

    <script src="/node_modules/ng2-bs3-modal/bundles/ng2-bs3-modal.js"></script>

**Comment by Flea505** [comment]
Thanks! I'll give it a go on Monday. 

Felicia Chavez
Tempus Nova
Cell 505 401 4415
Google Voice 505-386-0168
Felicia.chavez@tempusnova.com
